playbooks/ansible-test-network-integration-base/files/bootstrap-trendmicro.deepsec.deepsec.yaml

```diff
@@ -16,3 +16,85 @@
         - ds_agent
         - dsm_s
       become: true
+
+    - block:
+      - name: Get a Session ID
+        uri:
+          url: "https://{{ ansible_host }}:443/rest/authentication/login/primary"
+          method: POST
+          body: "{{ lookup('file','roles/tm-deepsec-aws-setup/cred.json') }}"
+          body_format: form-urlencoded
+          return_content: yes
+          body_format: json
+          validate_certs: no
+        register: session_id
+
+      - name: Get a request ID
+        uri:
+          url: "https://{{ ansible_host }}:443/api/sessions"
+          method: POST
+          body: "{{ lookup('file','roles/tm-deepsec-aws-setup/cred_session.json') }}"
+          body_format: form-urlencoded
+          return_content: yes
+          body_format: json
+          validate_certs: no
+          headers:
+            Content-Type: application/json
+            cookie: "sID={{ session_id['content'] }}"
+            api-version: v1
+        register: request_id
+
+      - name: Create and get the API KEY
+        uri:
+          url: "https://{{ ansible_host }}:443/api/apikeys"
+          method: POST
+          body: "{{ lookup('file','roles/tm-deepsec-aws-setup/cred_apiKeys.json') }}"
+          body_format: form-urlencoded
+          return_content: yes
+          body_format: json
+          validate_certs: no
+          headers:
+            Content-Type: application/json
+            cookie: "{{ request_id['set_cookie'] }}"
+            rID: "{{ request_id['json']['RID'] }}"
+            api-version: v1
+        register: api_key_response
+
+      - name: Create temp file with API Key
+        copy:
+          dest: "/tmp/deepsec_api_key.txt"
+          content: "{{ api_key_response['json']['secretKey'] }}"
+        delegate_to: 127.0.0.1
```

Comment:
I don't think this is working as expected, you might want to, at the bottem. Create a new play, for the controller and write the file